### PR TITLE
Fix: tuned condition of welcome message appearance

### PR DESCRIPTION
**Pull request by Leo310**
Resolves: corona-school/project-user#520

### CODE REVIEW

File: src/widgets/ImportantInformation.tsx [hunk 1]
```diff
@@ -115,7 +118,8 @@ const ImportantInformation: React.FC<Props> = ({ variant }) => {
         (student?.canCreateCourse?.reason === 'not-instructor' && student.canRequestMatch?.reason === 'not-tutor')
     )
         infos.push({ label: 'kennenlernen', btnfn: [() => window.open(process.env.REACT_APP_SCREENING_URL)], lang: {} });
-    if (pupil && !pupil?.firstMatchRequest) infos.push({ label: 'willkommen', btnfn: [() => navigate('/group'), () => navigate('/matching')], lang: {} });
+    if (pupil?.firstMatchRequest !== null && !pupil?.firstMatchRequest && pupil?.subcoursesJoined && pupil?.subcoursesJoined.length == 0)
```
Comment (Jonasdoubleyou) [inline src/widgets/ImportantInformation.tsx:121]: How can firstMatchRequest be not null but false? Also this won't be set for accounts which requested a match before the timestamp was introduced, so better also check matches

Comment (Leo310) [reply src/widgets/ImportantInformation.tsx:121]: not sure but on the test students, firstMatchRequest also wasn't set. So I thought I would be on the safe side if I just check for null

Comment (Jonasdoubleyou) [reply src/widgets/ImportantInformation.tsx:121]: Yeah but then !firstMatchRequest should be enough no?

Comment (Leo310) [reply src/widgets/ImportantInformation.tsx:121]: oh true

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**